website/content/docs/nia/usage/run-ha.mdx

```diff
@@ -0,0 +1,184 @@
+---
+layout: docs
+page_title: Run Consul-Terraform-Sync with High Availability
+description: >-
+  Improve network automation resiliency by enabling high availability for Consul-Terraform-Sync. HA enables persistent task and event data so that CTS functions as expected during a failover event.
+---
+
+# Run Consul-Terraform-Sync with High Availability
+
+This topic describes how to run Consul-Terraform-Sync (CTS) configured for high availability. High availability is an enterprise capability that ensures that all changes to Consul that occur during a failover transition are processed and that CTS continues to operate as expected.
+
+## Introduction
+
+A network always has exactly one instance of the CTS cluster that is the designated leader. The leader is responsible for monitoring and running tasks. If the leader fails, CTS triggers the following process if it is configured for high availability:
+
+1. The CTS cluster promotes a new leader from the pool of followers in the network.
+1. The new leader begins running all existing tasks in `once-mode` in order to process changes that occurred during the failover transition period. In this mode, CTS runs all existing tasks one time. 
+1. The new leader logs any errors that occur during `once-mode` operation and the new leader continues to monitor Consul for changes. 
+
+In a standard configuration, CTS exits if errors occur when the CTS instance runs tasks in `once-mode`. In a high availability configuration, CTS logs the errors and continues to operate without interruption. 
+
+The following diagram shows operating state when high availability is enabled:
+
+![Consul-Terraform-Sync architecture configured for high availability before a shutdown event](/img/nia/cts-ha-before.svg)
```

Comment:
+1 Diagram. I think from the diagram, it's implicit that the tasks are executed only by the leader instead of divided up. Do you think it would be helpful to mention that explicitly?